backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/ResultUploadErrorSource.java

```diff
@@ -0,0 +1,6 @@
+package gov.cdc.usds.simplereport.db.model.auxiliary;
+
+public enum ResultUploadErrorSource {
+  REPORTSTREAM,
+  SIMPLEREPORT,
```

Comment:
nit picks: change to 
REPORT_STREAM
SIMPLE_REPORT

Reply:
Sure! Done in [4e11732](https://github.com/CDCgov/prime-simplereport/pull/5971/commits/4e11732c96f67f10b616f96abf45822d163bcbff).